plugins/woocommerce-admin/client/tasks/tasks.tsx

```diff
@@ -150,10 +121,9 @@ export const Tasks: React.FC< TasksProps > = ( { query } ) => {
 				.map( ( taskList: TaskListType ) => {
 					const { id, isHidden, isToggleable } = taskList;
 
-					const TaskListComponent = getTaskListComponent( id );
```

Comment:
@moon0326 I found that the `Things to do next` header disappears. 🤔 Could you take a look? I think this might be because we need to use `TaskList` component for the extended task list.  Thanks!

![Screen Shot 2022-07-28 at 11 02 59](https://user-images.githubusercontent.com/4344253/181411344-3dceafee-9be6-4cfe-bed9-ee802695eb04.png)

Reply:
Thank you for catching it @chihsuan 👍 

Updated the code to use the two col components when task list id is setup in https://github.com/woocommerce/woocommerce/pull/34084/commits/c3f27483df01091433aa75f573eca977f2c31432

Could you take a look at it again please? 🙏

Reply:
Thank you. I can see the header now!